extensions/podman/src/extension.ts

```diff
@@ -1603,7 +1611,7 @@ export async function calcPodmanMachineSetting(podmanConfiguration: PodmanConfig
 
   if (isWindows()) {
     const isPodmanHyperv_Env = process.env.CONTAINERS_MACHINE_PROVIDER === 'hyperv';
-    const isPodmanHyperv_Config = await podmanConfiguration.matchRegexpInContainersConfig(/provider\s*=\s*"hyperv"/);
+    const isPodmanHyperv_Config = await new HyperVConfiguration(podmanConfiguration).isHyperVEnabled();
```

Comment:
Just tried on Windows and i saw that when calling the `isHyperVEnabled()` the configuration file which is used to verify if hyperv is enabled is `.config/containers/containers.conf`. However if i check my filesystem i see that i don't have any file in that location but i have one in `C:\ProgramData\containers\containers.conf.d\...`. Is it missing something?

Reply:
PTAL @vzhukovs

Reply:
@lstocchi @benoitf the logic of processing configuration files is the following and completely repeats the logic that is implemented in podman:

0) creating empty stack of file list
1) we are looking to env variable `CONTAINERS_CONF` and if exists, push to the stack
2) look for the default containers config file, which is empty in windows, so this step skips
3) look for overriden container config file, on windows it is located in `%PROGRAMDATA%\containers\containers.conf`, push to the stack tail
3.1) look for the directory `%PROGRAMDATA%\containers\containers.conf.d` and find any `*.conf` files in it, push them to the stack tail
4) look for user config in `%APPDATA%\containers\containers.conf`, push to the stack tail
4.1) look for the directory `%APPDATA%\containers\containers.conf.d`, do the same as in #3.1
5) create an empty config object and iterating from the stack head to the stack tail by merging every configuration with created config object. merging - means, that if in user configuration there is a parameter, e.g. hyperv = true, it will override system one. 

can you provide more details, which config files do you have in system?